jsx_test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Card from "./card";
+
+export default class CardList extends React.Component {
+  constructor(props)
+  {
+    super(props);
+  }
+
+  render () {
+    var card_list = this.props.deck_list.map(function(card)
```

Comment:
'deck_list' is missing in props validation                                     react/prop-types<br>'deck_list.map' is missing in props validation                                 react/prop-types<br>Opening curly brace does not appear on the same line as controlling statement  brace-style